src/partials/template.html

```diff
@@ -0,0 +1,20 @@
+<head>
+    <title></title>
+    <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
+    <style type="text/css">
```

Comment:
Perhaps we can extract css definitions in a css file

Reply:
done